tests/test_mapping.py

```diff
@@ -8,7 +8,7 @@ def test_point(self):
         self.assertEqual(m['type'], 'Point')
         self.assertEqual(m['coordinates'], (0.0, 0.0))
         
-    def test_empty_polygon():
+    def test_empty_polygon(self):
         """Empty polygons will round trip without error"""
         self.assertIsNotNone(mapping(Polygon()))
```

Comment:
The only thing missing now is `from shapely.geometry import Polygon` at the top of the file.